net-misc/httpie/httpie-2.3.0.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
+PYTHON_COMPAT=( python3_{6,7,8} )
```

Comment:
No Python 3.9 support yet?

Reply:
According to upstream's `setup.py`, any Python version >= 3.6 will do. I don't have a test environment available, but I guess adding 3.9 anyway won't hurt.